.swift-version

```diff
@@ -0,0 +1 @@
+2.2-dev
```

Comment:
It's for `chswift` which relies on the output of `swift --version` which doesn't include the snapshot version.